app/lib/methods/helpers/normalizeMessage.ts

```diff
@@ -11,17 +14,17 @@ function normalizeAttachments(msg) {
 		if (att.ts) {
 			att.ts = moment(att.ts).toDate();
 		}
-		att = normalizeAttachments(att);
+		att = normalizeAttachments(att as IMessage & IAttachment);
 		return att;
 	});
 	return msg;
 }
 
-export default msg => {
+export default (msg: any): IMessage | null => {
```

Comment:
`any` ?

Reply:
Diego specified me to do this because this isn't the same as an `IMessage`, it just gets converted into one.